src/components/learner-credit-management/tests/BudgetDetailPage.test.jsx

```diff
@@ -701,4 +701,69 @@ describe('<BudgetDetailPage />', () => {
 
     expect(screen.getByText('loading budget activity overview')).toBeInTheDocument();
   });
+
+  it('displays bulk actions with remind when row selected and state allocated', () => {
+    useSubsidyAccessPolicy.mockReturnValue({
+      isInitialLoading: false,
+      data: {
+        uuid: 'a52e6548-649f-4576-b73f-c5c2bee25e9c',
+        policyType: 'AssignedLearnerCreditAccessPolicy',
+        isAssignable: true,
+      },
+    });
+    const mockLearnerEmail = 'edx@example.com';
+    const mockCourseKey = 'edX+DemoX';
```

Comment:
As ESLint is pointing out above, these variables already defined globally for the file? I see these same variables created on line 60 and 61 on `master`, available to use within any test cases in the file.

Reply:
I think this is a rebasing issue 👍